jupyterhub/alembic/versions/0eee8c825d24_added_properties_column.py

```diff
@@ -0,0 +1,42 @@
+"""added properties column
+
+Revision ID: 0eee8c825d24
+Revises: 833da8570507
+Create Date: 2021-09-15 14:04:09.067024
+
+"""
+# revision identifiers, used by Alembic.
+revision = '0eee8c825d24'
+down_revision = '833da8570507'
+branch_labels = None
+depends_on = None
+
+from alembic import op
+import sqlalchemy as sa
+from jupyterhub.orm import JSONDict
+
+
+def upgrade():
+
+    try:
+        op.add_column("groups", sa.Column("properties", JSONDict))
+        print("Added column!")
+    except:
```

Comment:
I think this upgrade shouldn't be allowed to fail. Let's also remove the debug-prints.

`4dc2d5a8c53c_user_options.py` is an example of what adding a column should look like.

Reply:
I structured the alembic revision so that it corresponds to the user_options one.